recipe/recipe.yaml

```diff
@@ -37,71 +33,97 @@ source:
         - fix-argminmax.patch
 
 build:
-  number: 0
-  skip: not (match(python, python_min ~ ".*") and is_abi3)
-  python:
-    version_independent: true
+  number: 1
 
-requirements:
-  build:
-    - if: build_platform != target_platform
-      then:
+outputs:
+  - package:
+      name: polars
+    build:
+      script: echo "No build step is needed"
+      # Since this meta package is noarch: generic, we only need to build it
+      # for one target platform (does not matter which one)
+      skip: target_platform != 'linux-64'
+      noarch: generic
+      string: h${{ hash }}_${{ polars_variant | replace('-', '_') }}_${{ build }}
+      variant:
+        down_prioritize_variant: ${{ 0 if polars_variant == 'default' else 1 }}
+    requirements:
+      run:
+        - ${{ pin_subpackage('polars-' ~ polars_variant, lower_bound='x.x.x', upper_bound='x.x.x') }}
```

Comment:
Why not?

```suggestion
        - ${{ pin_subpackage('polars-' ~ polars_variant, exact=True) }}
```

Reply:
Because @pavelzw talked me into making the meta package noarch (;)), which means we only build it once per variant, which means that one build of the meta package must be compatible with the three real packages for that variant, which means you cant pin the build string exactly.

Reply:
Fair enough, sounds reasonable. Thanks for the explanation :smile: